.github/workflows/nomad-demo.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Run demo performance tests on Nomad cluster"
+
+on:
+  workflow_dispatch:
+    inputs:
+      holochain_bin_url:
+        description: "The URL to download the `holochain` binary from"
+        default: "https://github.com/holochain/holochain/releases/download/holochain-0.5.6/holochain-go-pion-unstable-x86_64-unknown-linux-gnu"
+        required: false
+      commit_hash:
+        description: "The commit hash to checkout before running the tests"
+        required: false
+  pull_request:
+    paths:
+      - ".github/workflows/nomad*.yaml"
+      - ".github/actions/nomad.yaml"
```

Comment:
Is this right, or do we want to also run the demo nomad jobs when the scenarios change?